src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -109,19 +109,24 @@ public function normalize($object, $format = null, array $context = array())
                 $attributeValue = $maxDepthHandler($attributeValue, $object, $attribute, $format, $context);
             }
 
+            // Adding support for serialized-name
+            array_key_exists($attribute, $attributesMetadata) && null !== $attributesMetadata[$attribute]->getSerializedName() ?
```

Comment:
I don't know if this syntax work, it's not documented: http://us2.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary

Anyway, IMHO it will be easier to read if written as:

```suggestion
            $serializedName = (array_key_exists($attribute, $attributesMetadata) && null !== $attributesMetadata[$attribute]->getSerializedName()) ?
                              $attributesMetadata[$attribute]->getSerializedName() :
                              $attribute;
```

(and it can be expanded in multiple lines to avoid one long line with 2 expressions)